vivisect/qt/funcgraph.py

```diff
@@ -118,6 +122,7 @@ def renderMemory(self, va, size, cb):
         [%d, %d]
         ''' % (selector, selector, va, size)
         runner = functools.partial(self._renderMemoryCallback, cb)
+        logger.log(MIRE, "renderMemory(%r, %r) %r", va, cb, runner)
```

Comment:
this debug message is one of "event sequencing" since Viv's event model has often been confusing.
it's very helpful when focusing on one funcgraph window.

Reply:
i'll remove it once we've done the Mem/GraphView merge, k?